aconfig_test.go

```diff
@@ -752,6 +750,45 @@ func TestCustomNames(t *testing.T) {
 	}
 }
 
+func TestDontGenerateTags(t *testing.T) {
```

Comment:
Nit: if feature is called `DoNot*`, maybe the test can be consistent with that to make it easier to find the relevant test in future.